### PR TITLE
Arianna detector 200

### PR DESCRIPTION
**Pull request by lpyras**
ARIANNA Detector 200 with 4 LPDAs pointing downwards, 3 LPDAs pointing upwards, plus deep channel.

### CODE REVIEW

File: NuRadioReco/detector/ARIANNA/arianna_detector_200.json [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+            "decommission_time": "{TinyDate}:2038-01-01T00:00:00",
+            "station_id": 101
+        },
+        "2": {
+            "adc_id": null,
+            "adc_n_samples": 256,
+            "adc_nbits": null,
+            "adc_sampling_frequency": 1.0,
+            "adc_time_delay": null,
+            "amp_reference_measurement": "300-03-01",
```
Comment (cg-laser) [inline NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: can you remove the amp reference measurement fields? This is a future station where we don't know the amps yet

File: NuRadioReco/detector/ARIANNA/arianna_detector_200.json [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+            "amp_reference_measurement": "300-03-03",
+            "amp_type": "300",
+            "ant_comment": "close to center, pointing downwards (4/4)",
+            "ant_deployment_time": "{TinyDate}:2018-12-05T00:00:00",
+            "ant_orientation_phi": 0.0,
+            "ant_orientation_theta": 180.0,
+            "ant_position_x": 2.1,
+            "ant_position_y": -2.1,
+            "ant_position_z": -3,
+            "ant_rotation_phi": -45.0,
```
Comment (cg-laser) [inline NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: the antennas should point inwards, we define the rotation of the lpdas by "where the connector points to" So it needs to be -45+180

Comment (cg-laser) [reply NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: same for the first channel

Comment (cg-laser) [reply NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: and all upward LPDAs

File: NuRadioReco/detector/ARIANNA/arianna_detector_200.json (hunk 1 above)
Comment (lpyras) [reply NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: okay, i set them to "null"

File: NuRadioReco/detector/ARIANNA/arianna_detector_200.json (hunk 2 above)
Comment (lpyras) [reply NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: Question: At the moment all antennas pointing to the centre (see detector visualization)
newplot

How should i change them?

Comment (cg-laser) [reply NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: hmm that's weird. Maybe something in the visualization script is wrong. I'll have a look later.

Comment (cg-laser) [reply NuRadioReco/detector/ARIANNA/arianna_detector_200.json]: ok I finally figured out what was wrong. The zenith angle was not correctly defined. In our definition, the zenith angle is only allowed within [0deg, 180deg]. 0deg is pointing upwards (to the zenith), 180deg is pointing downwards. Having a negative zenith angle resulted in undefined behavior.

**Comment by anelles** [comment]
@lpyras, now that the pull request has been approved, you can go ahead and merge it. This is our typical work-flow. The person who makes the pull request should do the merge.

**Pull request merged**